compose/eidr-connect.yml

```diff
@@ -0,0 +1,27 @@
+eidr-connect-bsve.eha.io:
+  image: eidr-connect
+  container_name: eidr-connect.eha.io
+  environment:
```

Comment:
This should have a `ONE_WAY_SYNC_URL=https://eidr-connect.eha.io` variable so it pulls new events from our instance.